src/components/Calculator.js

```diff
@@ -1,46 +1,58 @@
-import React from 'react';
+/* eslint-disable */
+import React, { useState } from "react";
+import calculate from "../logic/calculate";
+
+const Calculator = () => {
+  const [obj, setObj] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
+
+  const handleClick = (buttonName) => {
+    setObj(calculate(obj, buttonName));
+  };
 
-function Calculator() {
   return (
-    <div className="calculator-container">
-      <div className="calculator-output">0</div>
+    <div className="calculator-contaier">
+      <div className="calculator-output">{obj.next || obj.total || 0}</div>
       <div className="calculator-wrapper">
         <div className="calculator-count">
           <div className="calculator-numbers">
-            <b>AC</b>
-            <b>+/-</b>
-            <b>%</b>
+            <button onClick={() => handleClick("AC")}>AC</button>
+            <button onClick={() => handleClick("+/-")}>+/-</button>
```

Comment:
- [ ] Great job for practicing the use of `useState`, to also practice using `props` and follow `DRY `principle it's better to create a `Button `component and pass the props we need without affecting the implementation of the calculator.